cluster-up/cluster/kind-k8s-sriov-1.17.0/OWNERS

```diff
@@ -0,0 +1,31 @@
+filters:
+  ".*":
+    reviewers:
```

Comment:
I believe that for reviewers, Prow would always try to prefer this file over the root one. For approvers, all root approvers would have rights here, but the ones listed below would be preferred.

Could we limit the reviewers only to people who recently worked on this subdirectory? And for approvers, filter out only those who recently worked on the network part as well.

Reply:
Done, thanks